SplitIt/app/src/main/java/project/cs495/splitit/MainActivity.java

```diff
@@ -455,6 +461,7 @@ private void animateFab(int position) {
             case 2:
                 fab_scan_receipt.setVisibility(View.INVISIBLE);
                 fab_plus.setVisibility(View.INVISIBLE);
+                fab_add_group.setVisibility(View.VISIBLE);
```

Comment:
Logan had mentioned something about this not actually being the correct button. Refer to GroupMe for that.